libphal/phal_exception.H

```diff
@@ -19,6 +19,7 @@ enum ERR_TYPE {
 	SBE_STATE_READ_FAIL,
 	SBE_STATE_WRITE_FAIL,
 	SBE_CHIPOP_NOT_ALLOWED,
+	SBE_INVALID_STATE,
```

Comment:
**Commit Message:**

> For the primary processor, there will be primary and backup
secondary seeprom hosting SBE code.

For the processor, there will be primary and backup secondary seeprom that contains SBE code and measurement.

> For any SBE access/chip-op failure, execute p10_extract_sbe_rc
HWP. p10_extract_sbe_rc HWP determines the failure type and
specifies the recovery action to be done by BMC.

It's not correct, right? The service processor uses a chip-ops to interact with the SBE to do something. If that chip-ops call failed then, we need to figure out the reason by collecting the SBE FFDC. If the SBE FFDC collection failed with SBEFIFO_SEC_HW_TIMEOUT then, we are assuming it that the SBE is in a bad state (dead state) and we need to extract SBE RC and see whether it's telling to recover the SBE.

Reply:
agree may be need to reword